packages/opentelemetry/src/spanExporter.ts

```diff
@@ -197,20 +154,15 @@ export class SentrySpanExporter {
         bucketEntry.spans.delete(span);
       }
     }
+    // Cancel a pending debounced flush, if there is one
+    this._debouncedFlush.cancel();
```

Comment:
Why do we need to do this manually? Why cannot we rely on the `debounce` helper? (probably just needs comments for people like myself)

Reply:
Why not rename the _actual_ `flush` to `_flush` and make `flush()` the debounced one then?

Reply:
we allow to force-flush manually, skipping the debounce - this is what the public `flush()` method should do, so I would not change this. we only debounce internally :)  I will add some more comments to hopefully make these things clearer too!